### PR TITLE
PGI completions and Agribalyse tweaks for Eco-Score

### PR DESCRIPTION
**Pull request by teolemon**
I added totransfer.txt which is a backup of a file I only have on my drive. It will take time to finish migrating it.

### CODE REVIEW

**Comment by teolemon** [comment]
Errors in the categories taxonomy definition:
ERROR - fr:vezelay already is a synonym of fr:bourgogne-vezelay for entry fr:bourgogne-vezelay - fr:vezelay cannot be mapped to entry fr:vezelay
ERROR - it:tuscia already is a synonym of it:colli-etruschi-viterbesi for entry it:colli-etruschi-viterbesi - it:tuscia cannot be mapped to entry it:tuscia
ERROR - it:riviera-ligure already is a synonym of it:riviera-ligure for entry it:riviera-ligure - it:olio-di-oliva-riviera-ligure cannot be mapped to entry fr:huile-d-olive-riviera-ligure
Errors in the categories taxonomy definition at lib/ProductOpener/Tags.pm line 993.

**Comment by teolemon** [comment]
Errors in the categories taxonomy definition:
ERROR - it:asparago-verde-di-altedo has an undefined parent

File: taxonomies/totransfer.txt [hunk 1]
```diff
@@ -0,0 +1,11140 @@
...
+
+
+
+<en:German, Austria, Switzerlandn fruit spirit
+de, de, fr,de,it:Bodensee Obstler
+protected_name_file_number:en: PGI-DE+AT+CH-02570
+protected_name_type:en: gi
+origins:en: en:Germany, Austria, Switzerland
+
+<en:German, Austria, Switzerlandn fruit spirit
```
Comment (stephanegigandet) [inline taxonomies/totransfer.txt:2191]: There are several broken parents with commas

Comment (teolemon) [reply taxonomies/totransfer.txt:2191]: thanks

**Review by stephanegigandet — CHANGES_REQUESTED** [review]
I have not reviewed the whole file, but there are many parents with commas and parenthesis
e.g. <en:French fruit, vegetables and cereals

Also the multilingual entries like this one won't work:

<en:Dutch, Belgium, German spirit drinks
nl, nl,fr, de:Genièvre aux fruits, Vruchtenjenever , Jenever met vruchten, Fruchtgenever
protected_name_file_number:en: PGI-NL+BE+DE-01994
protected_name_type:en: gi
origins:en: en:Netherlands, Belgium, Germany

File: taxonomies/totransfer.txt (hunk 1 above)
Comment (teolemon) [reply taxonomies/totransfer.txt:2191]: @stephanegigandet this one is totransfer.txt. It's normal. It's not transferred yet.

**Comment by teolemon** [comment]
@stephanegigandet this one is totransfer.txt. It's normal. It's not transferred yet.

**Review by stephanegigandet — APPROVED** [review]

**Pull request merged**